### ISSUE TITLE
Property 'download_backedup_asset' does not exist on type 'typeof utils'

### ISSUE THREAD

**Issue by mpimentel-xealth** [cloudinary/cloudinary_npm#556]
## Bug report for Cloudinary NPM SDK
Before proceeding, please update to latest version and test if the issue persists
✓

## Describe the bug in a sentence or two.
Typescript: `Property 'download_backedup_asset' does not exist on type 'typeof utils'`

## Issue Type (Can be multiple)
[ ] Build - Can’t install or import the SDK
[ ] Babel - Babel errors or cross browser issues
[ ] Performance - Performance issues
[ ] Behaviour - Functions aren’t working as expected (Such as generate URL)
[ ] Documentation - Inconsistency between the docs and behaviour
[✓] Incorrect Types - For typescript users who are having problems with our d.ts files
[ ] Other (Specify)

## Steps to reproduce
See https://github.com/mpimentel-xealth/cloudinary-type-issue/blob/main/index.ts

Note that it actually works: https://github.com/cloudinary/cloudinary_npm/blob/master/lib/utils/index.js#L1111 just throws type error in TS

## Error screenshots
N/A

## Browsers (if issue relates to UI, else ignore)
N/A

## Versions and Libraries (fill in the version numbers)
Cloudinary_NPM SDK version: 1.30.1
Node - v14.19.1
NPM - 6.14.16

## Config Files (Please paste the following files if possible)
```
{
  "name": "cloudinary-type-issue",
  "version": "1.0.0",
  "description": "",
  "main": "index.ts",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "repository": {
    "type": "git",
    "url": "git+https://github.com/mpimentel-xealth/cloudinary-type-issue.git"
  },
  "author": "",
  "license": "ISC",
  "bugs": {
    "url": "https://github.com/mpimentel-xealth/cloudinary-type-issue/issues"
  },
  "homepage": "https://github.com/mpimentel-xealth/cloudinary-type-issue#readme",
  "dependencies": {
    "cloudinary": "^1.30.1"
  }
}
```

## Repository
https://github.com/mpimentel-xealth/cloudinary-type-issue/blob/main/index.ts

Thanks!

**Comment by skalahasti-cloudinary** [comment]
Hi @mpimentel-xealth,

Thanks for reporting this issue. 
We have created an internal ticket to work on this. I will keep you posted as soon as the issue is fixed.

Best Regards,
Sree

**Comment by skalahasti-cloudinary** [comment]
Hi @mpimentel-xealth ,

This issue was fixed. Please confirm if it is working fine at your end now.

Best Regards,
Sree

**Comment by mpimentel-xealth** [comment]
Verified fixed on 1.31.0

Thank you!
